docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -1405,14 +1405,14 @@ Result:
 └─────────────────────┴─────────────────────┴──────────┘
 ```
 
-## today
+## today {#date_time_functions#today}
```

Comment:
Two `#` in one anchor is probably a mistake

Reply:
Yes, it is a mistake --> fixed.
Weird that CI did not complain.